hash/msx1_cass.xml

```diff
@@ -13524,6 +13712,18 @@ license:CC0-1.0
 		</part>
 	</software>
 
+	<software name="skydivera" cloneof="skydiver">
+		<description>Sky Diver (Japan, alt)</description>
+		<year>1984</year>
+		<publisher>SoftBank</publisher>
+		<info name="usage" value="Load with BLOAD&quot;CAS:&quot;,R."/>
+		<part name="cass1" interface="msx_cass">
+			<dataarea name="cass" size="11732">
+				<rom name="sky diver - hudson soft (alt).cas" size="11732" crc="30522dd2" sha1="96873d866b7989d76964db1aaedeec13a6c9568b"/>
+			</dataarea>
+		</part>
+	</software>
```

Comment:
Is this actually from SoftBank?  The title screen says Hudson Soft.  I believe at the time SoftBank had exclusive distribution rights to Hudson’s MSX software.

Reply:
It was indeed from Hudson Soft